src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_NullCoalescingOperator.cs

```diff
@@ -237,12 +237,12 @@ private BoundExpression GetConvertedLeftForNullCoalescingOperator(BoundExpressio
             // before performing the leftConversion.
             // See comments in Binder.BindNullCoalescingOperator referring to GetConvertedLeftForNullCoalescingOperator for more details.
 
-            if (!TypeSymbol.Equals(rewrittenLeftType, rewrittenResultType, TypeCompareKind.ConsiderEverything2) && rewrittenLeftType.IsNullableType())
+            if (!TypeSymbol.Equals(rewrittenLeftType, rewrittenResultType, TypeCompareKind.ConsiderEverything) && rewrittenLeftType.IsNullableType())
```

Comment:
>ConsiderEverything [](http://example.com/codeflow?start=91&length=18)

What analysis was performed to make this change? If this is just a mechanical change, we should undo it because the point of using ```ConsiderEverything2``` was to come back and think about what kind should actually be used.

Reply:
That is the point of `ConsiderEverything2`. I think `ConsiderEverything` is correct here but given this is not central to this PR, I'm reverting.

Reply:
> That is the point of ConsiderEverything2. I think ConsiderEverything is correct here 

This isn't that obvious to me. It looks like we are concern about identity convertibility between the types here. In cases like that ignoring everything (nullable annotations, dynamic vs. object, custom modifiers, ...) feels more appropriate. Changes like this, usually, should be accompanied by tests proving that the used option is appropriate and necessary.